example_dags/example_dymaic_task_template.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
Should we also add a sphinx page explaining Dynamic Task templates and listing the ones which are available, linking to this example?

Reply:
Yes, we will add documentation for it. just waiting for the 1st documentation PR related to example to land so that we minimise rework.

Reply:
Added, please let me know how it is looking now.

<img width="807" alt="Screenshot 2022-08-11 at 2 33 32 PM" src="https://user-images.githubusercontent.com/98807258/184099942-29988828-2749-4733-a1b5-3545548b7fe0.png">